### PR TITLE
Charter draft

### PR DESCRIPTION
**Pull request by markafoltz**
Creating a PR for commenting purposes.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -57,28 +69,62 @@ <h2 id="goals">
...
-      from Web pages. Specifically, the CG provides a venue for discussing and
-      incubating early proposals that aim to:
+      The <a href="https://www.w3.org/community/webscreens/">Second Screen
+      Community Group (CG)</a> explores the use of secondary screens from Web
+      pages. We propose that this Community Group incubate and develop
+      specifications and prototypes of network protocols that implement the
+      <a href="https://w3c.github.io/presentation-api/">Presentation API</a>.
+    </p>
+    <p>
+      This work has three primary purposes:
```
Comment (avayvod) [inline index.html:80]: It seems odd that Remote Playback API is not mentioned here but is part of the goals below.

File: index.html [hunk 2]
```diff
@@ -100,36 +146,126 @@ <h2 id="background">
...
-      TBD
+      The following tasks are in scope for the work of the community group.
+    </p>
+    <ol>
+      <li>The specification of a set of network protocols that allows one user
+      agent to use the Presentation API to discover and present Web content on
+      another user agent, when both user agents are connected to the same Local
+      Area Network (LAN).
+      </li>
+      <li>Protocols that implement of all of the interfaces, attributes,
```
Comment (avayvod) [inline index.html]: nit: s/of all of/all of

File: index.html [hunk 3]
```diff
@@ -100,36 +146,126 @@ <h2 id="background">
...
+      methods, algorithms, and implementation requirements of the Presentation
+      API are in scope. This includes protocols that preserve the privacy and
+      security of data exchanged between the controlling and receiving user
+      agents.
+      </li>
+      <li>The 2-UA mode of the Presentation API is in scope. In 2-UA mode, the
+      receiving user agent renders the presented document independently and
+      messages between documents are exchanged via the network.
+      </li>
+      <li>Protocols that implement of all of the interfaces, attributes,
```
Comment (avayvod) [inline index.html]: ditto

File: index.html [hunk 4]
```diff
@@ -100,36 +146,126 @@ <h2 id="background">
...
     <h2 id="deliverables">
       Deliverables
     </h2>
+    <p>
+      The Community Group will deliver the following.
+    </p>
+    <ol>
+      <li>The <a href="#specifications">specifications</a> listed below.
+      </li>
+      <li>A prototype implementation those specifications, published on GitHub
```
Comment (avayvod) [inline index.html]: s/implementation/implementation of/

File: index.html [hunk 5]
```diff
@@ -138,26 +274,52 @@ <h3 id="specifications">
       section</strong>.
     </p>
     <p>
+      The Community Group will produce the following specifications.  The
+      protocols described by these specifications are designed to meet
+      the <a href="requirements.md">functional and security requirements</a> of
+      the Presentation API and the Remote Playback API.
+    </p>
+    <ol>
+      <li>A specification controlling and receiving user agents,
```
Comment (avayvod) [inline index.html]: s/specification/specification for

File: requirements.md [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+
+## <a name="spec-ddc"></a>Presentation Display Discovery and Connection requirements
+
+The following requirements apply to the proposed specification, *Presentation
+Display Discovery and Connection*.
+
+### <a name="REQ-D01"></a>REQ-D01: Presentation Display Availability
+
+- A controller shall be able to discover the presence of a receiver connected to
+the same IPv4 or IPv6 subnet and reachable by IP multicast.
```
Comment (avayvod) [inline requirements.md:26]: What about other ways to discover/connect to devices like nearby presence, NFC tap and so on?

File: index.html (hunk 1 above)
Comment (markafoltz) [inline index.html:80]: Added mention here.

File: index.html (hunk 2 above)
Comment (markafoltz) [inline index.html]: Fixed

File: index.html (hunk 3 above)
Comment (markafoltz) [inline index.html]: Ditto

File: index.html (hunk 4 above)
Comment (markafoltz) [inline index.html]: Done

File: index.html (hunk 5 above)
Comment (markafoltz) [inline index.html]: Done

File: requirements.md (hunk 6 above)
Comment (markafoltz) [inline requirements.md:26]: I'm adding some deliverables to the charter for non-normative reports on other technologies for discovering nearby screens (bluetooth, NFC).  I can add a note here that requirements for these technologies are to be determined (and will be embedded within those reports).

**Pull request merged**